src/app/store.ts

```diff
@@ -1,10 +1,16 @@
 import { configureStore, ThunkAction, Action } from '@reduxjs/toolkit';
 // eslint-disable-next-line import/no-cycle
 import counterReducer from '../features/counter/counterSlice';
+import usersSlice from '../features/usersSlice';
```

Comment:
The task requires to create a slice for 'author' and 'selectedPost', but it seems they are missing in your implementation. Make sure to create and import these slices.

Reply:
selectedPost should be a separate file ? correct ?
But the author is the user, right?